lib/nve/nve-main-page.dart

```diff
@@ -403,26 +606,63 @@ class GreenPage extends State<MyStatefulGreenPage> {
   @override
   Widget build(BuildContext ctxt) {
     return new Scaffold(
-        backgroundColor: Colors.green,
-        body: Column(
-            mainAxisAlignment: MainAxisAlignment.center,
-            crossAxisAlignment: CrossAxisAlignment.center,
-            children: <Widget>[
-              Center(
-                child: Text("Boundary :"),
-              ),
-              Center(
-                child: _setBoundary(),
-              ),
-              Center(
-                child: Text("Enable alerts :"),
-              ),
-              Center(
-                child: Checkbox(
-                  value: settings.enableAlerts,
-                  onChanged: (value) => _setAlerts(value),
-                ),
-              )
-            ]));
+      backgroundColor: Colors.green,
+      body: Stack(
+        children: <Widget>[
+          Material(
+              clipBehavior: Clip.antiAlias,
+              animationDuration: Duration(seconds: 1),
+              color: Colors.transparent,
+              child: Center(
+                  child: Container(
+                      height: 200,
+                      width: 400,
+                      child: Card(
+                        child: Stack(children: <Widget>[
+                          Positioned(
+                            width: 350,
+                            top: 20,
+                            left: 20,
+                            child: _setBoundary(),
+                          ),
+                          Positioned(
+                            bottom: 50,
+                            left: 100,
+                            child: Text(
+                              "Enable alerts :",
+                              style: TextStyle(fontWeight: FontWeight.bold),
+                            ),
+                          ),
+                          Positioned(
+                            bottom: 35,
+                            right: 100,
+                            child: Checkbox(
+                              value: settings.enableAlerts,
+                              onChanged: (value) => _setAlerts(value),
+                            ),
+                          )
+                        ]),
+                      )))),
+        ],
+      ),
+    );
+  }
+}
+
+class MyCustomClipper extends CustomClipper<Path> {
+  @override
+  Path getClip(Size size) {
+    var path = new Path();
+    path.moveTo(0, size.height * 0.5);
+    path.quadraticBezierTo(
+        size.width / 1.8, size.height / 1.6, size.width, size.height * 0.5);
+    path.lineTo(size.width, 0);
+    path.lineTo(0, 0);
+    return path;
+  }
+
+  @override
+  bool shouldReclip(CustomClipper<Path> oldClipper) {
+    return false;
   }
 }
```

Comment:
this file becomes very long. can you try to extract each page in a separated file ?